react-ts.json

```diff
@@ -0,0 +1,76 @@
+{
+  "extends": [
+    "airbnb",
+    "plugin:prettier/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": 8,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "modules": true,
+      "jsx": true,
+      "legacyDecorators": true
+    }
+  },
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true,
+    "jest/globals": true
+  },
+  "globals": {
+    "window": true,
+    "document": true
+  },
+  "plugins": [
+    "babel",
+    "jest"
+  ],
+  "rules": {
+    "no-use-before-define": 0,
+    "no-empty": 0,
+    "no-restricted-syntax": 0,
+    "no-await-in-loop": 0,
+    "no-param-reassign": 0,
+    "no-confusing-arrow": 0,
+    "no-nested-ternary": 0,
+    "camelcase": 0,
+    "arrow-parens": [
+      "error",
+      "as-needed"
+    ],
+    "import/extensions": 0,
+    "import/no-unresolved": 0,
+    "react/jsx-filename-extension": 0,
+    "react/require-default-props": 0,
+    "react/sort-comp": 0,
+    "react/prop-types": 0,
+    "react/jsx-closing-bracket-location": 0,
+    "react/forbid-prop-types": 0,
+    "react/destructuring-assignment": 0,
+    "react/no-access-state-in-setstate": 0,
+    "react/jsx-one-expression-per-line": 0,
+    "react/jsx-wrap-multilines": 0,
+    "react/jsx-indent": 0,
+    "jsx-a11y/anchor-is-valid": 0,
+    "jsx-a11y/click-events-have-key-events": 0,
+    "jsx-a11y/no-static-element-interactions": 0,
+    "jsx-a11y/label-has-for": 0,
+    "jsx-a11y/label-has-associated-control": 0,
+    "babel/new-cap": 1,
+    "jest/no-identical-title": "error",
+    "prettier/prettier": [
```

Comment:
I think we should separate prettier from eslint so we can have a .prettierrc so that editors/others can use it.

Reply:
all major editors have linting plugins for `eslint` which provides a superset of prettier's linting functionality, is there a specific reason to have editors interface with prettier directly?

Reply:
I can't think of any specific reason.  We can continue using eslint to handle code styling.